libs/js/banglejs/Bangle_showLauncher.js

```diff
@@ -1,12 +1,32 @@
 (function() {
-  var l = require("Storage").list(/\.info$/).map(file => {
-    var app = require("Storage").readJSON(file,1);
-    if (app && app.type == "launch") return app;
-  }).find(x=>x);
-  if (l) {
+  let s = require("Storage").readJSON("setting.json",1)||{};
+  let launcherApp = s.launcher;
+
+  if (!launcherApp) {
+    launcherApp = require("Storage").list(/\.info$/)
+      .map(file => {
+        const app = require("Storage").readJSON(file,1);
+        if (app && app.type == "launch") {
+          return app;
+        }
+      })
+      .filter(x=>x)
+      .sort((a, b) => a.sortorder - b.sortorder)[0].src;
+    print(launcherApp);
```

Comment:
Is this left over from debugging?

Reply:
Yes it was. Fixed another oversight in the new commit as well.